src/utils/facilities/formatFacilityOptionsToServer.spec.js

```diff
@@ -29,6 +31,32 @@ describe('utils/facilities/formatFacilityOptionsToServer', function() {
     expect(options).to.deep.equal(expectedOptions);
   });
 
+  it('provides an object that includes the `include_cost_centers` that the API service can understand', function() {
```

Comment:
Maybe missing a word? This didn't read quite fluidly to me.

Reply:
👍 

Added a word that hopefully helps